packages/backend/src/graphql/schema.graphql

```diff
@@ -860,6 +861,19 @@ type TemplateStep {
   sampleUrl: String # specific to template e.g. form or tile link
   sampleUrlDescription: String # differs for each step e.g. view a sample form
   parameters: JSONObject
+  tileTemplateStepData: [JSONObject] # the column names must match the tile that was created above!
+}
+
+# This is for creation of tile for a template
+type TileTemplateData {
+  name: String!
+  columns: [TileColumnData!]!
+  rowData: [JSONObject]
+}
+
+type TileColumnData {
+  name: String!
+  position: Int!
 }
```

Comment:
used for?

Reply:
cus every tiletemplate data format is fixed, so i didn't put it as a JSONObject?